agents/packages/ethlisbon/skills/price_prophet/behaviours.py

```diff
@@ -99,7 +99,7 @@ def get_strict(self, key: str) -> Any:
         """Get strict from SynchronizedData"""
         return self.synchronized_data.db.get_strict(key)
 
-    @functools.cached_property
+    @property
```

Comment:
why the change?

Reply:
for some reason this worked but it didnt previously